server/src/game/role/jailor.rs

```diff
@@ -98,8 +98,8 @@ impl RoleStateImpl for Jailor {
                 AbilitySelection::new_boolean(false),
                 actor_ref.ability_deactivated_from_death(game) ||
                 Detained::is_detained(game, actor_ref) || 
-                self.executions_remaining == 0 ||
-                game.day_number() <= 1 ||
+                self.executions_remaining <= 0 ||
```

Comment:
Not sure why this was changed -- or why clippy isn't telling you not to do this

Reply:
For context: It's impossible for this number to be less than zero, so writing the conditional this way is unnecessary and could be confusing to the reader

Reply:
You do this in other places in this PR, too